scripts/publish-apps

```diff
@@ -11,7 +11,7 @@ set -e;
 cd repos/aragon-apps;
 
 LERNA_EXEC_APPS="npx lerna exec --scope=@aragon/apps-finance --scope=@aragon/apps-voting --scope=@aragon/apps-token-manager --concurrency=1 --stream"
```

Comment:
Might as well also add the Survey app :smile:

Reply:
The survey app failed on travis with `VotingPanel.js:3:15: Cannot resolve dependency 'BN.js'`. I had no problem locally.